frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.tsx

```diff
@@ -43,7 +44,14 @@ const LinkButton = styled.span`
 `;
 
 export const LandReviewPage: React.FC<any> = (props: IReviewProps) => {
+  const withNameSpace: Function = useCallback(
```

Comment:
What's the value of this?  Is it just to make line 54 shorter?  Seems like a util that would be more performant and less duplication if you just did this `applyNamespace(props.nameSpace, 'financials.0.netbook.year')`.

Reply:
Yeah and to keep consistent with the rest of our code, I know the name spaces can get a bit nested for certain fields (buildings, financials) so this helps control it through the parent components. But yes I agree, I'll make a util and include in next PR - we use this a lot!